lib/views/accordion.js

```diff
@@ -0,0 +1,104 @@
+import React, {Fragment} from 'react';
+import PropTypes from 'prop-types';
+
+import {autobind} from '../helpers';
+
+export default class Accordion extends React.Component {
+  static propTypes = {
+    leftTitle: PropTypes.string.isRequired,
+    rightTitle: PropTypes.string,
+    results: PropTypes.arrayOf(PropTypes.any).isRequired,
```

Comment:
do we not know the shape of the results?

Reply:
Accordion is meant to be a reusable widget; in general, `results` could be anything that the `children` render-prop can interpret. Of course, in practice, these are always `Issueish` instances, because the only time render an Accordion is when we have a list of those. It felt wrong to constrain it here, though.

Reply:
👍

Reply:
... so this is still true here, but I see I got lazy in the PropTypes for `IssueishListView`, where we _do_ know that they're an array of `Issueishes`. I just pushed a commit that tightens up that constraint there. 😅